### PR TITLE
[JavaScript]  Challenge 3 (ReadyForMerge)

### PR DESCRIPTION
**Pull request by Zirhc86**

### CODE REVIEW

File: challenge_3/javascript/Zirhc86/src/majority-element.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+var array = [2,2,3,7,5,7,7,7,4,7,2,7,4,5,6,7,7,8,6,7,7,8,10,12,29,30,19,10,7,7,7,7,7,7,7,7,7];
+
+// Finds majority element using the Boyer-Moore majority vote algorithm.
+function findMajority(arr){
+	var count = 0, currentMajority;
+
+	for(var i =0; i<arr.length; i++){
+		if(count ==0){
```
Comment (halogenandtoast) [inline challenge_3/javascript/Zirhc86/src/majority-element.js]: Could probably use a space after the `==`

File: challenge_3/javascript/Zirhc86/src/majority-element.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+var array = [2,2,3,7,5,7,7,7,4,7,2,7,4,5,6,7,7,8,6,7,7,8,10,12,29,30,19,10,7,7,7,7,7,7,7,7,7];
+
+// Finds majority element using the Boyer-Moore majority vote algorithm.
+function findMajority(arr){
+	var count = 0, currentMajority;
+
+	for(var i =0; i<arr.length; i++){
```
Comment (halogenandtoast) [inline challenge_3/javascript/Zirhc86/src/majority-element.js]: Could probably use a space after the `=`, or remove the space before, your choice.

File: challenge_3/javascript/Zirhc86/src/majority-element.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+			count++;
+		}
+		else{
+			count--;
+		}
+	}
+
+	count=0;
+
+	for(var i =0; i<arr.length; i++){
```
Comment (halogenandtoast) [inline challenge_3/javascript/Zirhc86/src/majority-element.js]: Could probably use a space after the `=`, or remove the space before, your choice.

**Comment by Zirhc86** [comment]
Fixed the spacing on assignments/operators, and added explanation for the Boyer-Moore voting algorithm. I'll keep the forms idea in mind for future challenges.

**Comment by jacktraf** [comment]
Looks good to me man, I would go so far as to change the title from (pending) to (ReadyForMerge) since I approved and halogenandtoast only suggested change to spacing, and you did that.
Anyone else have any objections?

**Pull request merged**